x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/header.tsx

```diff
@@ -38,19 +33,19 @@ export default function Header() {
       description={
         <FormattedMessage
           id="xpack.datasetQuality.appDescription"
-          defaultMessage="Monitor the data set quality for {logsPattern} data streams that follow the {ecsNamingSchemeLink}."
+          defaultMessage="Monitor the data set quality for {logsPattern} data streams that follow the {dsNamingSchemeLink}."
           values={{
             logsPattern: <EuiCode>{DEFAULT_LOGS_DATA_VIEW}</EuiCode>,
-            ecsNamingSchemeLink: (
+            dsNamingSchemeLink: (
               <EuiLink
-                data-test-subj="datasetQualityAppDescriptionEcsNamingSchemeLink"
-                href={docLinks.links.ecs.dataStreams}
+                data-test-subj="datasetQualityAppDescriptionDsNamingSchemeLink"
+                href="https://ela.st/data-stream-naming-scheme"
                 target="_blank"
                 rel="noopener"
               >
                 <FormattedMessage
-                  id="xpack.datasetQuality.appDescription.ecsNamingSchemeLinkText"
-                  defaultMessage="ECS naming scheme"
+                  id="xpack.datasetQuality.appDescription.dsNamingSchemeLinkText"
+                  defaultMessage="Data Stream naming scheme"
```

Comment:
nit: Not sure if "Stream" should be capitalized.

Reply:
I think it can be lowercase as in [Data Stream naming scheme](https://www.elastic.co/guide/en/fleet/7.17/data-streams.html#data-streams-naming-scheme), but let's merge it in to make sure it makes it in time before FF, we can always change it if needed

Reply:
Yeah, it is lowercase.